### PR TITLE
no-ribbon slide modifier added

### PR DESCRIPTION
**Pull request by uyouthe**
I added the modifier that could be useful for ribbon removal at just one slide.

Usage:
`<section class="slide no-ribbon"> ... </section>`

Use the `no-ribbon` modifier to hide the ribbon at just one slide.

It happens just like when you type
`<section class="slide black"> ... </section>`
or
`<section class="slide white"> ... </section>`
but `no-ribbon` modifier doesn't affect the slide in any other way.

### CODE REVIEW

File: styles/slide/modifiers/no-ribbon.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// ------------------------------------------------------------------
+// No ribbon
+// ------------------------------------------------------------------
+
+&.white {
```
Comment (pepelsbey) [inline styles/slide/modifiers/no-ribbon.scss]: Wrong class name

**Comment by uyouthe** [comment]
Oh my god, sorry for that

**Comment by pepelsbey** [comment]
Maybe `empty` or `clean` would be better? Proposed `no-ribbon` is too robotic to my taste, like `is-large` or `no-js`. What actually happens here: it removes slide number. By giving it a bit more universal name we could also use it in Material and other themes.

```html
<section class="slide clean">
```

What do you think?

**Comment by pepelsbey** [comment]
FYI: that’s why it took me so long to review this PR, I didn’t like something about it, but only now I realized what exactly.

**Comment by shvaikalesh** [comment]
+1 for universal name, but `clean` is not descriptive enough IMO. `no-slide-number`?

**Comment by pepelsbey** [comment]
Sorry it took so long to deal with this PR, but I’m really grateful for your idea. I’ve just released Ribbon and Material themes updates featuring a new `clear` slide type which removes page numbers and progress bar. It makes it even modular: instead of depending on white/black slides you can just have a `slide black clear` or `slide clear cover`.

Check it yourself [on Ribbon’s demo page](https://shwr.me/shower/themes/ribbon/#20).

**Pull request closed without merging**